src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -45,7 +60,7 @@
             pathString = pathString.Add('/' + shellSettings.RequestUrlPrefix);
         }
 
-        return $"{Context.Request.Scheme}://{hostString.Value + pathString.Value}";
+        return $"{Context.Request.Scheme}://{hostString + pathString.Value}";
```

Comment:
Here it is wrong to assume that the hostnames are all using {Context.Request.Scheme}